website/blog/2024-10-23-release-0.76-new-architecture.md

```diff
@@ -0,0 +1,221 @@
+We want the tooling you use to debug React across all platforms to be reliable, familiar, simple, and cohesive. React Native DevTools is an out-of-the-box experience that delivers on these principles — familiar browser-aligned developer tools that are deeply integrated with React Native.
+Key features:
+
+- **Familiar, web-aligned tooling** — A fully featured debugger based on Chrome DevTools, providing reliable breakpoints, watch values, step through debugging, stack inspection, and a rich JavaScript console. These core features now work reliably and across reloads.
+- **An improved and integrated version of React DevTools** — The built-in React Components Inspector and Profiler are powerful tools to inspect and understand your React app, and work just as they do on the web. Component highlighting is faster and more reliable than before.
+- **Improved UX** — See a new Paused in Debugger overlay, making it clear when your app is paused on a breakpoint. Warnings in LogBox are now displayed as a summary, and hidden when DevTools is attached.
+- **Fixed reconnection behavior** — JavaScript breakpoints now work reliably across full reloads and when DevTools is disconnected and reattached. DevTools can even reconnect to the same app after a native rebuild, and displays the app and device name in the window title.
+- **Instant launch** — React Native DevTools is ready by default with zero config. Open from the in-app Dev Menu or via <kbd>j</kbd> to debug in the CLI server, which now supports multiple emulators and devices.
+
+React Native DevTools is fundamentally different from our previous debugging options, including being different from the Experimental Debugger experience first shipped in 0.73. It switches to a new backend debugging stack that we’ve rebuilt from the ground up over the last year, in partnership with the Hermes team. This means that compatibility with previous tooling [has changed](https://github.com/react-native-community/discussions-and-proposals/discussions/819#:~:text=announcement%20talk%20%F0%9F%8E%AC.-,Compatibility,-React%20Native%20DevTools), and you should also expect a much more reliable experience end-to-end. These foundational improvements will support the addition of future features that we're keen to bring to users, such as the Performance and Network panels.
+Phasing out logs in Metro
+In our next release, we’re also removing forwarded logs in Metro. We are doing this to align with modern browser tooling and to remove old, non-CDP, debugging integrations. Streamed application logs are not a pattern in web bundlers or in native build tools, and these are neither well-formatted nor searchable. Instead, please use React Native DevTools and its fully featured Console panel. See more under our [FAQs](https://github.com/react-native-community/discussions-and-proposals/discussions/819#:~:text=point%20is%20deprecated.-,FAQs,-React%20Native%20DevTools).
+
+**Links**
```

Comment:
```suggestion
To learn more about the new React Native DevTools, here some useful resources:
```

Reply:
nit: I'd rather keep this terse. As replied on the GDoc, I don't think this strongly establishes any new convention. Happy with another short heading, e.g. "Resources".